react.gradle

```diff
@@ -132,7 +132,7 @@ afterEvaluate {
                 }
 
                 // Workaround for Android Gradle Plugin 3.2+ new asset directory
-                into ("merged_assets/${targetPath}/merge${targetName}Assets/out") {
+                into ("merged_assets/${targetName}/merge${targetName}Assets/out") {
```

Comment:
Thers is already an ongoing PR https://github.com/facebook/react-native/pull/21782, and the capitalized `targetName` seems incorrect.